src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -88,6 +92,28 @@ public ResponseEntity<Object> createNewSubmission(
         }
     }
 
+    @PutMapping("/{overseas_entity_id}")
+    public ResponseEntity<Object> updateSubmission(
+            @RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+            @PathVariable(OVERSEAS_ENTITY_ID_KEY) String submissionId,
+            @RequestParam(SAVE_POINT) SavePoint savePoint,
```

Comment:
Now intending to remove this parameter and associated enum, following discussion. Will endeavour to decide how much validation to apply based on data present in the DTO body.